mappings/net/minecraft/util/registry/RegistryEntry.mapping

```diff
@@ -1,26 +1,115 @@
 CLASS net/minecraft/class_6880 net/minecraft/util/registry/RegistryEntry
+	COMMENT An object holding a value that can be registered in a registry. In most cases, the
+	COMMENT value is already registered in a registry ("reference entry"), hence the name;
+	COMMENT however, it is possible to create a registry entry by direct reference
+	COMMENT ("direct entry"). This is useful for data packs, as they can define
+	COMMENT one-time use values directly without having to register them every time.
+	COMMENT
+	COMMENT <p>Registry entries do not define {@code equals} method. Instead, compare the result
+	COMMENT of {@link #getKeyOrValue}.
+	COMMENT
+	COMMENT <p>Reference registry entries also hold their {@linkplain TagKey tags}. For more
+	COMMENT information on type-specific behaviors, including "intrusive" and "stand-alone"
+	COMMENT reference registry entries, see the respective class documentations.
+	COMMENT
+	COMMENT <p>A registry entry is sometimes referred to as a "holder" in error messages.
+	COMMENT
+	COMMENT @see RegistryEntry.Direct
+	COMMENT @see RegistryEntry.Reference
+	COMMENT @see Registry#entryOf
+	COMMENT @see Registry#getEntry
+	METHOD comp_349 ()Ljava/lang/Object;
+		COMMENT {@return the value of the registry entry}
```

Comment:
Hmm, is it really useful to have a doc comment for record accessors since the methods can't be seen?